Makefile

```diff
@@ -16,15 +16,27 @@ build_xiplot: xiplot/.gitignore
 	pip install . && \
 	python3 -m build
 
-xiplot: install_xiplot build_xiplot
+build_plugins: xiplot/.gitignore
+	cd xiplot && \
+	rm -f plugins/*.whl && \
+	cd test_plugin && \
+	rm -rf dist && \
+	pip install build && \
+	python3 -m build && \
```

Comment:
Here you need a `mkdir -p ../plugins && \` otherwise `../plugins` becomes a file